app/models/Tag.java

```diff
@@ -120,6 +85,27 @@ public long getNofCompletedExercises(User currentUser) {
         ).sum();
     }
 
+
+    public static Tag processCreate(String tagName, boolean isMainTag, User user) {
+        Tag tag = Tag.find().all().stream()
```

Comment:
Suech das mit Ebean query (Tag.find().where()...), nöd mit de Stream API -> so musch zersch alles us de DB is java suuge